mne/viz/utils.py

```diff
@@ -261,10 +263,10 @@ def _get_help_text(params):
     """Aux function for customizing help dialogs text."""
     text, text2 = list(), list()
 
-    text.append(u'\u2190 : \n')
-    text.append(u'\u2192 : \n')
-    text.append(u'\u2193 : \n')
-    text.append(u'\u2191 : \n')
+    text.append(u'\u2190 : \n')  # left arrow
+    text.append(u'\u2192 : \n')  # right arrow
+    text.append(u'\u2193 : \n')  # down arrow
+    text.append(u'\u2191 : \n')  # up arrow
```

Comment:
why not actually put the unicode characters in the file?

https://unicode-table.com/en/sets/arrows-symbols/